src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -445,13 +480,13 @@ export class MatrixRTCSession extends TypedEventEmitter<
         public memberships: CallMembership[],
         /**
          * The session description is used to define the exact session this object is tracking.
-         * A session is distinct from another session if one of those properties differ: `roomSubset.roomId`, `sessionDescription.application`, `sessionDescription.id`.
+         * A session is distinct from another session if one of those properties differ: `roomSubset.roomId`, `slotDescription.application`, `slotDescription.id`.
          */
-        public readonly sessionDescription: SessionDescription,
+        public readonly slotDescription: SlotDescription,
```

Comment:
The doc comment should be updated to no longer mention "session description." I like this language from the MSC:

> a virtual address where participants are allowed to meet